src/main/java/com/synopsys/integration/detect/lifecycle/boot/decision/ProductDecider.java

```diff
@@ -1,12 +1,14 @@
 package com.synopsys.integration.detect.lifecycle.boot.decision;
 
+import java.util.List;
 import java.util.Optional;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import com.synopsys.integration.detect.configuration.connection.BlackDuckConnectionDetails;
 import com.synopsys.integration.detect.configuration.enumeration.BlackduckScanMode;
+import com.synopsys.integration.detect.configuration.enumeration.DetectTool;
```

Comment:
changes in this class seem unnecessary? Code was maybe removed but imports were left?

Reply:
very likely... will investigate

Reply:
yes removing now